bot/exts/previewing/preview.py

```diff
@@ -37,16 +39,19 @@ async def server_icon(
         icon = (await download_image(file_url)).resize(ICON_SIZE)
         icon = add_background(icon, Preview.background_color(mode.lower()))  # type: ignore
 
-        with Image.open(ICON_TEMPLATES.format(mode=mode)) as template:
-            preview = Image.new("RGBA", template.size)
-            for position in ICON_POSITIONS:
-                preview.paste(icon, position)
+        def preview():
```

Comment:
```suggestion
        def _preview():
```
So it doesn't have the same name as the `preview` variable.